xarray/tests/__init__.py

```diff
@@ -45,6 +45,12 @@
 
 
 def _importorskip(modname, minversion=None):
+    """
+    This is deprecated - instead use pytest.importorskip. For example:
+
+    matplotlib = pytest.importorskip('matplotlib', minversion='1.9.0')
+
+    """
```

Comment:
I don't this can work until https://github.com/pytest-dev/pytest/issues/568 is fixed.